### ISSUE TITLE
Loottables are buggy so spaceship#0 gets generated

### ISSUE THREAD

**Issue by ProsperCraft** [mcenderdragon/Futurepack-API#377]
When you click a block like a block of iron with the replicator, if you do this several times fast, it dupes the block 4-5 times.

**Comment by mcenderdragon** [comment]
what replicator ?

**Comment by mcenderdragon** [comment]
If it is fp:spaceship with meta 0, then it is a debug item, wich can crash worlds, destroy buildings and is absolut evil. If yor home/ship/world gets deleted/moved/transported to another dimension/turned upside down/rotaated then this is not my fault. 

wow, I should add this as safety disclaimer to the tooltip of the item.

**Comment by ProsperCraft** [comment]
![construtionreplicator](https://user-images.githubusercontent.com/4427684/36188393-187932ce-1109-11e8-8dad-fee308ed23be.png)

The Construction Replicator from your mod.

**Comment by ProsperCraft** [comment]
This item is lootable in your dungeons :)

**Comment by mcenderdragon** [comment]
yeah this item should be called "vanillas lootables are bugy as hell". Because sometimes they can not set the item damge (no one knows why) so this item ends up the loot chests. You should not be able to get this item, as most of the debug features end up in this item.

**Comment by ProsperCraft** [comment]
My players are still getting these after the world reset...

https://i.gyazo.com/2b90d331eaac42b8c9b96c76a39d9f9d.png

**Comment by mcenderdragon** [comment]
You can check the log for something like: unable to set item metadata. Vanillas loottables tend to break :/
